### PR TITLE
Add the yaml env load

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Add environment variables load by yaml file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -218,3 +218,4 @@ empty_test_environment_variables.py
 /cloud_governance/policy/send_mail.py
 cloudsensei/.env.txt
 .vscode
+cloud_governance/main/env.yaml
```
Comment (ebattat) [inline .gitignore:221]: Where did u put this file

**Review by ebattat — COMMENTED** [review]
Great initiative !!!!
need to update the readme file how to run podman using -v yaml file

File: .gitignore (hunk 1 above)
Comment (athiruma) [reply .gitignore:221]: inside the main folder

Comment (ebattat) [reply .gitignore:221]: Sorry I dont see it

Comment (athiruma) [reply .gitignore:221]: I added to .gitignore file that's why it is hidden.

Comment (ebattat) [reply .gitignore:221]: @athiruma ,
`cloud_governance/main/env.yaml`: remove from .gitignore and it should be filled with default/empty values
`cloud_governance/main/test_env.yaml`: add to .gitignore and it should be filled with real values

Comment (athiruma) [reply .gitignore]: No need to maintain 2 files, we can use test_env.yaml. In the readme, we will mention creating a copy of
`cp test_env.yaml env_yaml` and change appropriate values.

File: cloud_governance/main/test_env.yaml [hunk 2]
```diff
@@ -0,0 +1,167 @@
+policy: ""
```
Comment (ebattat) [inline cloud_governance/main/test_env.yaml:1]: I think it should be env.yaml

**Review by ebattat — COMMENTED** [review]
Pls add to readme file how to run the container using env.yaml instead of passing environment variable to the container

File: cloud_governance/main/test_env.yaml (hunk 2 above)
Comment (athiruma) [reply cloud_governance/main/test_env.yaml:1]: The one should copy to env.yaml and pass only defined values.

**Review by ebattat — COMMENTED** [review]
@athiruma, do you want to update readme documentation with yaml option ?

**Review by ebattat — CHANGES_REQUESTED** [review]
@athiruma, Can you pls add e2e test that using yaml file ?

**Comment by athiruma** [comment]
> @athiruma, Can you pls add e2e test that using yaml file ?

Added

File: cloud_governance/main/environment_variables.py [hunk 3]
```diff
@@ -4,18 +4,63 @@
...
+            except FileNotFoundError:
+                pass
+
+    def load_from_yaml(self, ):
+        """
+        Load values from yaml file as attributes of this class.
+        Will never override existing attributes.
+        """
+
+        for yaml_file_path in (os.path.join(os.path.curdir, 'env.yaml'), self.DEFAULT_CONF_PATH):
```
Comment (athiruma) [inline cloud_governance/main/environment_variables.py:45]: searching file in current `env.yaml` && `DEFAULT_CONF_PATH=/tmp/env.yaml`

**Review by ebattat — APPROVED** [review]
/LGTM

**Review by ebattat — APPROVED** [review]
/LGTM

File: .github/workflows/Build.yml [hunk 4]
```diff
@@ -384,7 +384,15 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
         run: |
-          sudo podman run --rm --name cloud-governance -e policy=${{ matrix.policy }} -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -e AWS_DEFAULT_REGION=${{ matrix.region }} -e dry_run=yes -e policy_output=s3://${{ secrets.BUCKET }}/test/${{ matrix.region }} -e log_level=INFO ${{ secrets.QUAY_PUBLIC_CLOUD_GOVERNANCE_REPOSITORY }}
```
Comment (ebattat) [inline .github/workflows/Build.yml:387]: @athiruma, I think we should run both yaml and env variabale, WDYT ?

Comment (athiruma) [reply .github/workflows/Build.yml:387]: yeah it helps.

Comment (ebattat) [reply .github/workflows/Build.yml:387]: Pls open new PR and double the e2e test yaml + env
Total should be 104 tests

**Pull request merged**